src/util/MultiMap.ts

```diff
@@ -1,22 +1,10 @@
-export class MultiMap<K, V> {
-  private map = new Map<K, V[]>();
-
-  add(key: K, value: V): void {
-    const all = this.map.get(key);
-    if (all) {
-      if (!all.includes(value)) {
-        all.push(value);
-      }
-    } else {
-      this.map.set(key, [value]);
+export function multiMapAdd<K, V>(map: Map<K, V[]>, key: K, value: V) : void {
```

Comment:
What do you think of naming the file `src/util/map.ts` so it can be used for all sorts of map helpers in the future, and maybe the function `addOne`?